test/e2e-common.sh

```diff
@@ -547,6 +548,25 @@ function test_setup() {
   fi
 }
 
+# Waits until the given service has some endpoints.
+# Parameters: $1 - namespace.
+#             $2 - service name.
+function wait_until_service_has_endpoints() {
+  echo -n "Waiting until service $2 in namespace $1 has an external address (IP/hostname)"
```

Comment:
can you log time here, on success and on failure to see how long it took?

Reply:
It's not an issue here, we're still failing the same way.